quickstart.rst

```diff
@@ -275,7 +275,7 @@ Here, the allocated ID for the job is immediately echoed to stdout.
 
 .. code-block:: console
 
-  $ flux job list
+  $ flux jobs
     JOBID           STATE   USERID    PRI   T_SUBMIT
     640671547392    R       58985     16    2019-10-22T16:27:02Z
     1045388328960   R       58985     16    2019-10-22T16:27:26Z
```

Comment:
The output of the `flux jobs` command also differs from `flux job list`, right?  Do you mind also updating that with the new format?

Reply:
Ah! Yes, you're right. That output is different. I went ahead and updated the output, squashed and force-pushed.

Reply:
Looking more closely, it looks like the `NAME` field from `flux jobs` gets truncated after a small number of characters. Is there a way to increase the length of that field to accommodate for a longer name?

Reply:
The default field width for the name is 10 characters. To increase the width of the field you'll have to override the default format for flux-jobs, e.g.

```console
$ flux mini submit --job-name=reallylongnameforsomereason sleep 300
1041965776896
$ flux jobs
             JOBID USER     NAME       STATE    NTASKS NNODES  RUNTIME RANKS
     1041965776896 grondo   reallylong RUN           1      1   1.588s 0
$ flux jobs --format="{name:<16}"
NAME            
reallylongnameforsomereason
```

However, that may be out-of-scope for a Quickstart document.